crypto/ec/ec_ameth.c

```diff
@@ -672,8 +635,13 @@ int ec_pkey_export_to(const EVP_PKEY *from, void *to_keydata,
     if (tmpl == NULL)
         return 0;
 
+    bnctx = BN_CTX_new_ex(libctx);
+    if (bnctx == NULL)
+        goto err;
+    BN_CTX_start(bnctx);
```

Comment:
Unfortunately because the param builder has shallow references I need to set up a BN_CTX so the BIGNUM values used in tmpl dont get trashed when the function ec_group_todata() returns.